jvm/controls/src/main/resources/css/add-plugin-dialog.css

```diff
@@ -16,10 +16,9 @@
  * You should have received a copy of the GNU General Public License
  * along with Orature.  If not, see <https://www.gnu.org/licenses/>.
  */
-@import "/css/root.css";
 
 .add-plugin-dialog {
-    -fx-background-color: -wa-white;
+    -fx-background-color: -wa-background;
```

Comment:
I would give the variable more specific names. In this case -wa-dialog-background